### PR TITLE
Add tests for gaLayers service

### PR DESCRIPTION
**Pull request by oterral**
[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_tests/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=190000.00&Y=660000.00&zoom=1&dev3d=true&debug)

No functional changes

### CODE REVIEW

File: test/karma-conf.mako.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = function(config) {
   basePath: '../prd',
 % else:
   basePath: '../src',
-% endif
+% endif;
```
Comment (gjn) [inline test/karma-conf.mako.js]: not sure about the semicolon here....

**Comment by gjn** [comment]
LGTM. Could not test 3D related stuff (I'm on limited hardware currently), and also no IE tests possible. Could you do it?

One small comment.

This PR conflicts a little with mine, but I'll correct once this is merged.

**Comment by gjn** [comment]
@loicgasser could you quickly test IE and 3D? Then let's get this in.

**Comment by oterral** [comment]
I can test the 3d and IE9

File: test/karma-conf.mako.js (hunk 1 above)
Comment (oterral) [inline test/karma-conf.mako.js]: yes strange I remove it

Comment (gjn) [inline test/karma-conf.mako.js]: there are more below

**Comment by oterral** [comment]
No bug found

**Pull request merged**